openfold/data/input_pipeline_multimer.py

```diff
@@ -21,19 +21,11 @@
     data_transforms_multimer,
 )
 
-
-def nonensembled_transform_fns(common_cfg, mode_cfg):
-    """Input pipeline data transformers that are not ensembled."""
-    transforms = [
-        data_transforms.cast_to_64bit_ints,
-        data_transforms_multimer.make_msa_profile,
-        data_transforms_multimer.create_target_feat,
-        data_transforms.make_atom14_masks,
-    ]
-
-    if mode_cfg.supervised:
+def grountruth_transforms_fns():
+        
+        transforms = []
```

Comment:
Define transforms as transforms = [data_transforms.make_atom14_masks....] etc instead of defining empty list and extending it